2 - ListsLogicAndIterations/Code/Dictionarys.cs

```diff
@@ -5,19 +5,40 @@ namespace ListsLogicAndIterations.Code
 {
 	public static class Dictionarys
 	{
-		public static Dictionary<Guid, decimal> Build(Guid key, decimal value)
-		{
-			throw new NotImplementedException();
+        public static Dictionary<Guid, decimal> Build(Guid key, decimal value)
+        {
+            Dictionary<Guid, decimal> dict = new Dictionary<Guid, decimal>
+            {
+                { key, value }
+            };
+
+            return dict;
+
+
+            //throw new NotImplementedException();
 		}
 
 		public static decimal GetValue(Dictionary<Guid, decimal> dict, Guid key)
 		{
-			throw new NotImplementedException();
+            decimal result = 0.0m;
+
+            try
+            {
+                result = dict[key];
+            }
+            catch (KeyNotFoundException) { };
+
+            return result;
```

Comment:
Though you can catch an exception here, dealing with exceptions is expensive in both CPU operations and code flow. The preferred way here is to use a built in method on the dictionary called `ContainsKey()` which will return true or false.

https://www.dotnetperls.com/containskey